Sources/CMakeLists.txt

```diff
@@ -15,21 +15,31 @@ add_subdirectory(CoreCommands)
 add_subdirectory(CrossCompilationDestinationsTool)
 add_subdirectory(DriverSupport)
 add_subdirectory(LLBuildManifest)
+add_subdirectory(PackageCollections)
+add_subdirectory(PackageCollectionsModel)
+add_subdirectory(packageCollectionsSigning)
+add_subdirectory(PackageCollectionsSigningLibc)
+add_subdirectory(PackageCollectionsTool)
 add_subdirectory(PackageDescription)
 add_subdirectory(PackageFingerprint)
 add_subdirectory(PackageGraph)
 add_subdirectory(PackageLoading)
+add_subdirectory(PackageMetadata)
 add_subdirectory(PackageModel)
 add_subdirectory(PackagePlugin)
 add_subdirectory(PackageRegistry)
+add_subdirectory(PackageRegistryTool)
 add_subdirectory(PackageSigning)
+add_subdirectory(SourceControl)
 add_subdirectory(SPMBuildCore)
 add_subdirectory(SPMLLBuild)
-add_subdirectory(SourceControl)
 add_subdirectory(swift-bootstrap)
 add_subdirectory(swift-build)
 add_subdirectory(swift-experimental-destination)
 add_subdirectory(swift-package)
+add_subdirectory(swift-package-collection)
+add_subdirectory(swift-package-manager)
```

Comment:
I think we should skip this one since it is the multi-tool for symlinking which we aren't using on Windows, right?

Reply:
Ah, excellent, that can shave some time on the build.  It doesn't matter if we build and install it - it can't get distributed without it being put into the installer manifest.

Reply:
Done